app/frontend/src/features/auth/password/CompleteResetPassword.tsx

```diff
@@ -32,20 +38,24 @@ export default function CompleteResetPassword() {
     }
   }, [completePasswordReset, resetToken]);
 
-  return isLoading ? (
-    <Typography variant="body1">{CHANGE_PASSWORD_PROGRESS}</Typography>
-  ) : isSuccess ? (
-    <>
-      <Alert severity="success">{CHANGE_PASSWORD_SUCCESS}</Alert>
-      <Typography variant="body1" component={Link} to={loginRoute}>
-        {CLICK_LOGIN}
-      </Typography>
-    </>
-  ) : (
-    error && (
-      <Alert severity="error">
-        {`${CHANGE_PASSWORD_ERROR}${error.message}`}
-      </Alert>
-    )
+  return (
+    <Container className={classes.standardContainer}>
+      {isLoading ? (
+        <CircularProgress />
+      ) : isSuccess ? (
+        <>
+          <Alert severity="success">{RESET_PASSWORD_SUCCESS}</Alert>
+          <MuiLink variant="body1" component={Link} to={loginRoute}>
+            {CLICK_LOGIN}
+          </MuiLink>
+        </>
+      ) : error ? (
+        <Alert severity="error">
+          {`${RESET_PASSWORD_ERROR}${error.message}`}
+        </Alert>
+      ) : (
+        ""
+      )}
```

Comment:
```suggestion
      ) : null}
```

Reply:
From memory, I originally had this but got type errors

Reply:
"fixed in 5.0" https://github.com/mui-org/material-ui/issues/21711